@navikt/core/react/src/form/textarea/Textarea.tsx

```diff
@@ -193,7 +194,17 @@ export const Textarea = forwardRef<HTMLTextAreaElement, TextareaProps>(
           aria-live="polite"
         >
           {showErrorMsg && (
-            <ErrorMessage size={size}>{props.error}</ErrorMessage>
+            <ErrorMessage
```

Comment:
Since this icon is now added to all Form errors, i think it would make sense to create a wrapper-component for "FormErrorMessage" that implements this. 

Or we should add the icon as a built-in for `ErrorMessage`, so that `icon` is a boolean-prop for turning it of/on so that everyone else can also use ut when wanted. The relevant CSS can then be added under `typography.css` instead of in textarea.css